build/extension_manifest.json

```diff
@@ -13,6 +13,13 @@
 			"*://tr.wikipedia.org/*",
 			"*://es.wikipedia.org/*"
 		],
+		"exclude_globs": [
+			"*://en.wikipedia.org/wiki/Special:*",
```

Comment:
Each of these probably also need `"*://en.wikipedia.org/*title=Special:*"` — don't think there's anything caught by that that shouldn't be.

Reply:
Added, with the translations.